android/app/src/main/java/com/maker/makerapp/MainActivity.java

```diff
@@ -4,7 +4,6 @@
 import com.facebook.react.ReactActivity;
 import com.facebook.react.ReactActivityDelegate;
 import com.facebook.react.ReactRootView;
-import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
```

Comment:
Do we not need this anymore?

Reply:
No. It's not needed with the latest reanimated